probe/endpoint/connection_tracker.go

```diff
@@ -0,0 +1,258 @@
+package endpoint
+
+import (
+	"strconv"
+
+	log "github.com/Sirupsen/logrus"
+	"github.com/weaveworks/scope/probe/endpoint/procspy"
+	"github.com/weaveworks/scope/probe/process"
+	"github.com/weaveworks/scope/report"
+)
+
+// connectionTrackerConfig are the config options for the endpoint tracker.
+type connectionTrackerConfig struct {
+	HostID       string
+	HostName     string
+	SpyProcs     bool
+	UseConntrack bool
```

Comment:
This is just for short-lived connection tracking, right? Shouldn't `UseConntrack` and `WalkProc` be merged?